results/reports/mochawesome.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
@Exactor96, do we need this file? Should there be a .gitignore setting to ignore all reports?

Reply:
and videos

Reply:
Done.